### DISCUSSION TITLE
Update/Edit Pack through Website

### DISCUSSION THREAD

**Discussion by PAPISOP** [BedrockTweaks/Files#188] [Ideas]
Such like how it is done in vanilla tweaks, we should be able to upload the pack to the website to automatically update the packs and the user can edit their pack choices and then they can download the new and updated pack

Some notes for backend: 
1. If there is some changes in category name or pack name and the pack uploaded to the website uses old name then it should update those names to latest one in packs.info and manifest description
2. If the pack has been moved to an other category then it should also update this in packs.info and manifest description
3. It should automatically remove the packs which have been removed
4. Bedrock Tweaks also makes a packs.info in the downloaded pack so the website can use this
5. Also add all the pack combinations used in the pack in packs.info

**Comment by SquatchHunter** [comment]
I believe DrAv has this planned out on their backend repo which we cannot see, I know there was an issue talking about similar functionality at one point
